Modules/Loadable/Markups/MRML/vtkCurveGenerator.cxx

```diff
@@ -628,12 +641,21 @@ int vtkCurveGenerator::GeneratePointsFromFunction(vtkPoints* inputPoints, vtkPoi
     previousPoint[0] = curvePoint[0];
     previousPoint[1] = curvePoint[1];
     previousPoint[2] = curvePoint[2];
+
+    // Calculate pedigree ID for point
```

Comment:
Please describe in the comment how it is calculated? It should be clear that pedigreeID is not the ID of the geometrically closest control point.